src/main/java/core/basesyntax/Bulldozer.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public class Bulldozer extends Machine {
+
+    @Override
+    public void doWork() {
+        System.out.println("Bulldozer started its work.");

```

Comment:
The message "Bulldozer started its work." could be more informative. According to the checklist, it should indicate the type of Machine that is working. Consider specifying that it is a Bulldozer, for example: "Bulldozer is now working."